builtin/providers/aws/resource_aws_instance.go

```diff
@@ -24,6 +24,9 @@ func resourceAwsInstance() *schema.Resource {
 		Update: resourceAwsInstanceUpdate,
 		Delete: resourceAwsInstanceDelete,
 
+		SchemaVersion: 1,
```

Comment:
Yeah I waffled on this - the fact that unset yields `0` made me gravitate towards keeping things consistently increasing by 1, but it'd be easy to change before we release 0.4.